build/build-image/cross/Dockerfile

```diff
@@ -17,6 +17,50 @@
 
 FROM golang:1.12.0
```

Comment:
the 1.12.1 tag is now published. do we still want to merge this workaround?

Reply:
@ixdy yes, This lets get the CI jobs get started quicker. We can revert things back on monday (and publish fresh images)

Reply:
https://github.com/kubernetes/kubernetes/issues/75372#issuecomment-473420258

Reply:
The PR merged a few hours ago, but you can also now pull the image, meaning this workaround isn't needed: 
```console
$ docker pull golang:1.12.1
1.12.1: Pulling from library/golang
22dbe790f715: Pull complete
0250231711a0: Pull complete
6fba9447437b: Pull complete
c2b4d327b352: Pull complete
619f4932b7ea: Pull complete
16e105463ffe: Pull complete
ed842bbe1793: Pull complete
Digest: sha256:8630e0f0e9a8187f54066ea81a9fe33a21ddbfbe88c7149fe2299d8cd989ad63
Status: Downloaded newer image for golang:1.12.1
```

Reply:
@ixdy i am happy for us to switch things back right now if you, @amwat and @BenTheElder are ok with it.

Reply:
build-cross v1.12.1-1 is already pushed with this workaround, so we should commit this to record the source of v1.12.1-1. that will also let CI soak begin for the weekend while a v1.12.1-2 without this workaround gets pushed.

Reply:
I'm up to switch it but it sounds like for now we want to let the CI soak then switch it for v1.12.1-2?
https://kubernetes.slack.com/archives/C2C40FMNF/p1552691032534800